fe1-web/src/features/rollCall/screens/ViewSingleRollCall.tsx

```diff
@@ -0,0 +1,350 @@
+import { CompositeScreenProps, useRoute } from '@react-navigation/core';
+import { useNavigation } from '@react-navigation/native';
```

Comment:
I think this file should in a future PR be split as in the case of the election

Reply:
Yes that's a great idea given the number of lines.